src/exercise/06.js

```diff
@@ -18,11 +18,38 @@ function UsernameForm({onSubmitUsername}) {
 
   // 🐨 make sure to associate the label to the input.
   // to do so, set the value of 'htmlFor' prop of the label to the id of input
+
+  const [error, setError] = useState(null);
```

Comment:
Just a random aside that setError doesn't appear to ever be set now (that we're enforcing no possible error state with the toLowerCase) so here and lines 51-53 of actually displaying the error are probably no longer necessary